blog.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles/stylesheet.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Irish+Grover&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d04e98713e.js" crossorigin="anonymous"></script>
+    <title>AdoptADog Blog</title>
+    <meta name="author" content="Valerie V. Mehring">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <div class="cart"><img src="images/cart-icon.svg"> </div>
+      <p>$246.90</p>
+      <a href="checkout.html" class="button">checkout</a>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><i class="fas fa-paw"></i></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><i class="fas fa-paw"></i></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h3>AdoptADog Blog</h3>
+      <article class="">
+        <img src="blog-1.jpg" alt="A man and dog enjoy the view of a canyon">
```

Comment:
Your blog images aren't displaying. You'll need to specify the folder the images are in. Such as `images/blog-1.jpg`